PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx

```diff
@@ -56,57 +57,63 @@
 #include "AliESDtrackCuts.h"
 #include "AliESDtrack.h"
 #include "AliAODInputHandler.h"
-//#include "AliAnalysisTaskSEHFTreeCreator.h"
+#include "AliAnalysisTaskSEHFTreeCreator.h"
```

Comment:
Please remove the inclusion of AliAnalysisTaskSEHFTreeCreatos since it is not needed and it would require the inclusion of the treeHF library in the dependencies of vertexingHF, which is not possible since treeHF already depends on vertexingHF.